<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookInfo</class>
 <widget class="QWidget" name="BookInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>808</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="picture">
                 <property name="minimumSize">
                  <size>
                   <width>240</width>
                   <height>320</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_3">
                   <item>
                    <widget class="QLabel" name="label">
                     <property name="maximumSize">
                      <size>
                       <width>40</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>标题：</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="title">
                     <property name="text">
                      <string>标题</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_8">
                   <item>
                    <widget class="QLabel" name="label_6">
                     <property name="maximumSize">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>页数:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="likes">
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>爱心数</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_9">
                   <item>
                    <widget class="QLabel" name="label_7">
                     <property name="maximumSize">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>收藏数</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="views">
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>收藏数</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_2">
                   <item>
                    <widget class="QLabel" name="updateTick">
                     <property name="enabled">
                      <bool>true</bool>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>80</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>80</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>TextLabel</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="isFinished">
                     <property name="minimumSize">
                      <size>
                       <width>80</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>80</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>完本</string>
                     </property>
                     <property name="textFormat">
                      <enum>Qt::AutoText</enum>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="startRead">
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>开始阅读</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTabWidget" name="tabWidget">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="epsWidget">
              <attribute name="title">
               <string>Tags</string>
              </attribute>
              <layout class="QGridLayout" name="epsLayout"/>
             </widget>
             <widget class="QWidget" name="commentWidget">
              <attribute name="title">
               <string>评论</string>
              </attribute>
              <layout class="QGridLayout" name="commentLayout"/>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>startRead</sender>
   <signal>clicked()</signal>
   <receiver>BookInfo</receiver>
   <slot>StartRead()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>607</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>AddDownload()</slot>
  <slot>AddFavority()</slot>
  <slot>StartRead()</slot>
 </slots>
</ui>
